### ISSUE TITLE
Change shapes

### ISSUE THREAD

**Issue by mhauru** [alan-turing-institute/AssurancePlatform#110]
Goal: Hexagon
Property claim: Node with round edges

**Comment by chrisdburr** [comment]
Context and system description: paralellogram variants
Evidential Claim: stadium
Evidential artefact (unchanged): cylinder

**Comment by chrisdburr** [comment]
## New Colour Palette

![palette 3](https://user-images.githubusercontent.com/63010234/166919809-6bc146af-6b42-4ad3-bc79-abc8f4341e1b.jpg)

**Comment by mhauru** [comment]
Do we want to visually differentiate between system and project property claims? If so, how?

**Comment by mhauru** [comment]
<img width="925" alt="image" src="https://user-images.githubusercontent.com/5229876/166936341-0cc6535e-7b6c-4d97-b41c-fc312baa22e0.png">

**Comment by chrisdburr** [comment]
Yes, let's have the evidential claim colour as the system property claim, and the project property claim can remain the same. 

Then, the evidential claim colour, can be switched to a much lighter version of the hue used currently for evidence.

**Comment by mhauru** [comment]
How about these for the colours?
![image](https://user-images.githubusercontent.com/5229876/167451337-a55976d1-8f07-484c-9708-1cc947206350.png)
